test/components/report/numeric-text-field_spec.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { expect } from 'chai'
+import { describe, it } from 'mocha'
+import { shallow } from 'enzyme'
+import NumericTextField from '../../../js/components/report/numeric-text-field'
+
+describe('<NumericTextField /> minValue prop', () => {
+  it('should not allow non-numeric values, and default to 0', () => {
+    const wrapper = shallow(<NumericTextField />)
+    wrapper.find('input').simulate('change', { target: { value: 'x' } })
+    wrapper.find('input').simulate('blur', {})
+    expect(wrapper.state('value')).to.equal(0)
+  })
+  it('not allow non-numeric values; default to 10 as we specify', () => {
+    const wrapper = shallow(<NumericTextField default={10} />)
+    wrapper.find('input').simulate('change', { target: { value: 'x' } })
+    wrapper.find('input').simulate('blur', {})
+    expect(wrapper.state('value')).to.equal(10)
+  })
+  it('will not allow negative numbers; default to default value', () => {
+    const wrapper = shallow(<NumericTextField default={10} />)
+    wrapper.find('input').simulate('change', { target: { value: '-1' } })
+    wrapper.find('input').simulate('blur', {})
+    expect(wrapper.state('value')).to.equal(10)
+  })
+  it('will not allow numbers less than default (10) as we specify', () => {
```

Comment:
I think the "it" description might be confusing. To me, this is testing that `it('will allow...'`